src/events/cd-booking-create-account.vue

```diff
@@ -118,13 +128,19 @@
         }
       },
       async submitAccount() {
-        return this.register()
+        return this.formatProfile()
+          .then(this.register)
           .then(this.addChildren)
           .then(this.joinDojo)
           .then(this.bookTickets);
       },
-      async register() {
+      formatProfile() {
         this.profile = StoreService.load(`booking-${this.eventId}-user`);
+        this.profile.email = this.email;
+        StoreService.save(`booking-${this.eventId}-user`, this.profile);
+        return Promise.resolve();
```

Comment:
Why? There's nothing async here. submitAccount should just be
```javascript
async submitAccount() {
  this.formatProfile();
  return this.register()
    .then(this.addChildren)
    .then(this.joinDojo)
    .then(this.bookTickets);
}
```

If you _really_ want to chain them, then just make this function async and the returned Promise is implicit.